public_html/templates/index.html

```diff
@@ -7,116 +7,136 @@
 			<span>
 				<a class="reviewer-link" href="{{ row.reviewed_by_url }}" target="_blank">{{ row.status_user }}</a>
 			</span><br>
-			<span class="reviewer-timestamp">{{ row.review_timestamp }}</span>
+			<time datetime="{{ row.review_timestamp }}" class="reviewer-timestamp">{{ row.review_timestamp }}</time>
 		</small>
 	</div>
 {% endmacro %}
 
 {% block records %}
 	{% if records is not empty %}
 		{% for row in records %}
-			<div class="row-container">
-				<div class="container-fluid">
-					<div class="row-div col-sm-3 page-div">
-						<b><a href='{{ row.page_link }}' target="_blank" class={{ row.page_dead ? 'text-danger' : '' }}>
-								{{ row.page_title }}
-							</a></b>
-					</div>
-					<div class="row-div col-sm-1 text-center diff-div">
-						<a href='{{ row.diff_link }}' target="_blank">Diff</a><br>
-						<small class="diff-div-timestamp">
-							<span>{{ row.diff_timestamp }}</span>
-						</small>
-					</div>
-					<div class="row-div col-sm-2 text-center report-div">
-						{% if row.editor %}
-							<a href='{{ row.editor }}'
-							   class={{ row.editor_page_dead ? 'text-danger' : '' }}>{{ row.editor }}</a><br>
-							<small>
-								<span><a href='{{ row.editor_talk }}'
-								         class='{{ row.editor_talk_dead ? 'text-danger' : '' }}'
-								         target="_blank">Talk</a> | </span>
-								<span><a href='{{ row.editor_contribs }}' target="_blank">Contributions</a> </span><br>
-								<span>Edit count: {{ row.editcount }}</span>
+			<article class="record">
```

Comment:
`<article>` is for a "self-contained composition" on the page, where it is independent of the other articles on the page. This usually goes inside a `<section>`